src/controllers/autoCompletion.controller.js

```diff
@@ -1,9 +1,42 @@
 const Query = require("../models/Query");
+const User = require("../models/User");
+const jwt = require("jsonwebtoken");
 
 exports.getAutoCompletions = async function (req, res, next) {
+  const accessToken = req.cookies.accessToken;
+  const MAXIMUM_AUTO_COMPLETIONS = 5;
+
+  let userData;
+
+  if (accessToken) {
+    userData = jwt.verify(accessToken, process.env.JWT_SECRET_KEY);
+  }
+
   const searchHistories = [];
   const { userInput } = req.query;
-  const MAXIMUM_AUTO_COMPLETIONS = 5;
+  const user = userData?.userId;
+
+  if (user) {
+    const foundUser = await User.findOne({ _id: user });
```

Comment:
특별히 다른 조작이 없는 쿼리라 lean() 붙여주시면 좋을 것 같습니다!
```suggestion
    const foundUser = await User.findOne({ _id: user }).lean();
```

Reply:
그렇네요! 추가하도록 하겠습니다!